### DISCUSSION TITLE
Answers practice exam

### DISCUSSION THREAD

**Discussion by jonasmeij** [tudelft/AE4353-Y24#30] [Exam] [answered]
Hi i was wondering where i can find the answers of the practice exam?

**Comment by yilun-wu** [comment, marked as answer]
We will publish it soon on Brightspace.

**Reply by yilun-wu** [reply]
released just now on Brightspace

**Reply by kmalkow** [reply]
Can you please show us the result that you got?

**Reply by jonasmeij** [reply]
Excuse me, i accidentally changed the number of epochs. Your solution works

**Reply by kmalkow** [reply]
I see, no worries. Always good to ask, glad it works!
